app/Http/Controllers/DashboardController.php

```diff
@@ -0,0 +1,65 @@
+<?php declare(strict_types=1);
+
+namespace App\Http\Controllers;
+
+use Auth;
+use App\Models\Task;
+use Illuminate\Database\Eloquent\Collection;
+
+use Illuminate\Http\{
+    Request,
+    JsonResponse
+};
+
+class DashboardController extends Controller
+{
+    /**
+     * get organization and user-related task stats
+     *
+     * @param Request $request
+     * @return JsonResponse
+     */
+    public function getOrgUserStats(Request $request): JsonResponse
+    {
+        $tasks = Task::all();
+        $taskStats = $this->getPriorityBasedTaskStats($tasks);
+
+        return response()->json([
+            'stats' => $taskStats
+        ]);
+    }
+
+    /**
+     * get array of tasks organized by priority
+     *
+     * @param Collection $tasks
+     * @return array
+     */
+    private function getPriorityBasedTaskStats(Collection $tasks): array
```

Comment:
Out of curiosity, what’s quicker, doing this count or a query using count?

Reply:
I'm guessing having multiple queries set up and ran to get specific counts would take longer